### ISSUE TITLE
Error Parsing Response for Current Streams

### ISSUE THREAD

**Issue by mistersoup3** [p-hueber/prefetcharr#36]
I am getting the following error in the prefetcharr logs when shows are being watched on Jellyfin:

Error:
2024-04-12T02:35:06.963994Z ERROR prefetcharr::process: Failed to process err=error decoding response body
Caused by:
    EOF while parsing a value at line 1 column 0

Jellyfin version 10.8.13 (served through docker [latest tag])
Prefetcharr version 0.6.0 (served through docker)

The error seems to appear when any show is being watched, and appears during the scheduled scan process. If no show is being watched, the error does not appear. Does not seem to matter whether it is within the 'remaining episodes' amount or not. Please let me know if there's any other info I can add!

**Comment by mistersoup3** [comment]
Adding snippet from the docker-compose below. A note on the networking, I have confirmed that it is connecting to the two services with this vpn network mode. I originally screwed it up and was getting connection errors in the logs, but that is no longer happening:

  prefetcharr:
    image: phueber/prefetcharr:latest
    container_name: prefetcharr
    depends_on:
      vpn:
        condition: service_healthy
      jellyfin-latest:
        condition: service_healthy
    network_mode: 'service:vpn'
    environment:
      # `jellyfin`, `emby` or `plex`
      - MEDIA_SERVER_TYPE=jellyfin
      # Jellyfin/Emby/Plex baseurl
      - MEDIA_SERVER_URL=http://localhost:8097
      # Jellyfin/Emby API key or plex server token
      - MEDIA_SERVER_API_KEY=${JELLYFIN_TOKEN}
      # Sonarr baseurl
      - SONARR_URL=http://localhost:7878
      # Sonarr API key
      - SONARR_API_KEY=${SONARR_TOKEN}
      # Logging directory
      - LOG_DIR=/log
      # Log level
      - RUST_LOG=prefetcharr=trace
      # Polling interval in seconds (10 minutes)
      - INTERVAL=600
      # Minimum remaining episodes before a search
      - REMAINING_EPISODES=4
    volumes:
      - ${DFLT_CONFIG_DIR}/prefetcharr:/log

**Comment by p-hueber** [comment]
@mistersoup3 thanks for reporting! Before I have a look, could you add your Sonarr version please?

**Comment by p-hueber** [comment]
I have to improve the error reporting for HTTP status codes and add some sort of early probing..
There's something wrong with your Sonarr settings/connections. It could be an invalid API key for example.

**Comment by mistersoup3** [comment]
Sonar Info:

![image](https://github.com/p-hueber/prefetcharr/assets/166840358/546e6fbe-fa74-488a-a82a-51ff8d7d2f32)

**Comment by mistersoup3** [comment]
@p-hueber It was a config issue, I used the wrong port .-.
I've switched to the correct one (8989) and am re-running. It is now giving the following error:

2024-04-12T16:46:58.044892Z ERROR prefetcharr::process: Failed to process err=series not found in Sonarr

Is it possible to see what series it is looking at? Based on the Sessions result, I would think it is looking for 'Futurama' which is definitely there.
